src/VirtualClient/VirtualClient.Main/profiles/PERF-CPU-HPLINPACK.json

```diff
@@ -0,0 +1,48 @@
+{
+  "Description": "High Performance Linkpack Performance workload",
+  "Metadata": {
+    "RecommendedMinimumExecutionTime": "00:01:00",
+    "SupportedPlatforms": "linux-x64,linux-arm64",
+    "SupportedOperatingSystems": "Ubuntu"
+  },
+  "Actions": [
+    {
+      "Type": "HPLinpackExecutor",
+      "Parameters": {
+        "Scenario": "ProcessorSpeed",
```

Comment:
In the scenario, could you include the block size and  problem size? Or do you have another way to distinguish the data? I don't see it in the parser

Reply:
I have added it in the metadata for each metric generated. I could also add them in scenario to be more clear.